### ISSUE TITLE
Rename TotalProfit2Criterion and TotalLossCriterion 

### ISSUE THREAD

**Issue by diffstorm** [ta4j/ta4j#569]
I offer renaming the following classes;

TotalLossCriterion to GrossLossCriterion
TotalProfit2Criterion to GrossProfitCriterion

Thanks.

**Comment by nimo23** [comment]
Yes, good idea. Can u rename these classes and make a PR?

**Comment by nimo23** [comment]
TotalLossCriterion is not GrossLossCriterion because TotalLossCriterion uses 

`Num loss = trade.getProfit();`

Either the javadoc for TotalLossCriterion is wrong:

> Calculates the gross loss of the given trade

should be 

> Calculates the total loss (without commissions) of the given trade.

or the calcuation is wrong.

It should be 

`Num loss = trade.getGrossProfit();`

instead of 

`Num loss = trade.getProfit();`

**Comment by nimo23** [comment]
I think we should use 

`Num loss = trade.getGrossProfit();`

for `TotalLossCriterion` and rename it to `GrossLossCriterion`.

**Comment by diffstorm** [comment]
I think, this line below fixes your confusion in the `TotalLossCriterion` which is `GrossLossCriterion` in finance terminology. 
`return loss.isNegative() ? loss : series.numOf(0);`

BTW, I renamed them but I don't have permission to push my new branch.

**Comment by diffstorm** [comment]
Please check my pull request.

If the removed classes were needed, we can create new classes as TotalProfitAmountCriterion and TotalLossAmountCriterion to satisfy the old ones, but these are commonly used as percentages.
